classes/spec/capitalize_decorator_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'spec_helper'
+
+describe Decorator do
+  before :each do
+    @capitalize_decorator = CapitalizeDecorator.new('Covey')
+  end
+
+  context 'correct_name' do
+    it 'Check if nameable exists' do
+      expect(@capitalize_decorator.nameable).to eq 'Covey'
+    end
+  end
+end
```

Comment:
- [ ] Your don't have a spec file that tests the `TrimmerDecorator` class. Kindly write tests for this class and its `correct_name` method. Remember that all classes and their methods should have unit tests for them to ensure that the methods work as expected. This will help you spot bugs in time if there are issues with the method implementation. 👍 😎